modules/legacy/variables.tf

```diff
@@ -99,14 +99,20 @@ variable "kms_key_name" {
   default     = null
 }
 
-variable "labels" {
```

Comment:
I assume most of these are just moving the variables around and we did not remove or add any variables when porting main modules to modules/legacy?

Reply:
I made a few changes based on the clean up proposals. The changes are:
- Added `enable_streaming_engine ` and `skip_wait_on_job_termination`
- Changed `ip_configuration` to `use_public_ips`
- Renamed `network_self_link ` to `network_name`
- Renamed `subnetwork_self_link ` to `subnetwork`
- Removed `zone`

I just realized that I forgot to update example modules to reflect those changes. I will update them accordingly.

Reply:
Updated. PTAL.

Reply:
@harrisonlimh Let's call out any renames/changes in the upgrade guide